__tests__/app.test.js

```diff
@@ -61,6 +61,74 @@ describe("GET api/reviews", () => {
   });
 });
 
+describe("GET api/reviews?queries", () => {
+  test("200: should respond with an array of objects, filtered by category value when passed a category query parameter", () => {
+    return request(app)
+    .get("/api/reviews?category=social_deduction")
+    .expect(200)
+    .then(({body: {reviews}}) => {
+      expect(reviews.length).toBe(11)
+      reviews.forEach((review) => {
+        expect(review.category).toBe("social deduction")
+      })
+    })
+  })
+  test("200: should respond with an array of objects, sorted by the column defined by the sort_by parameter", () => {
```

Comment:
I would mention that the default order is descending in your test description here